crates/uv/tests/pip_compile_scenarios.rs

```diff
@@ -70,11 +68,18 @@ fn requires_incompatible_python_version_compatible_override() -> Result<()> {
 
     // In addition to the standard filters, swap out package names for more realistic messages
     let mut filters = INSTA_FILTERS.to_vec();
-    filters.push((r"a-3f4ac9b2", "albatross"));
-    filters.push((r"-3f4ac9b2", ""));
+    filters.push((
+        r"requires-incompatible-python-version-compatible-override-a",
+        "albatross",
+    ));
+    filters.push((
+        r"requires-incompatible-python-version-compatible-override-",
+        "pkg-",
+    ));
```

Comment:
I'll simplify the filters in a follow-up, this is in an attempt to minimize the diff